_layouts/podcast.html

```diff
@@ -13,5 +13,11 @@
 
   <div class="content">
     {{ content | replace: '<!--break-->', '<a class="anchor" id="continue"></a>' }}
+
+    {% if page.date %}
+      <h3>Description</h3>
+
+      {{ page.summary }}
```

Comment:
Why are you using page.summary as the main content element?

Reply:
I think I have to implement the links and stuff inside the summary, so all podcast viewers display the show notes as well...